src/app/(school)/(reviews)/@header/professor/[slug]/loading.tsx

```diff
@@ -0,0 +1,21 @@
+import { GraduationCapIcon } from "@/common/components/CustomIcon";
+import { PageTitle } from "@/common/components/PageTitle";
+import { Skeleton } from "@/common/components/Skeleton";
+import { Tag } from "@/common/components/Tag";
+
+export default function Loading() {
+  return (
+    <div className="w-full">
+      <PageTitle
+        contentLeft={<GraduationCapIcon className="h-9 w-9" />}
+        contentRight={
+          <Tag contentLeft={<Skeleton className="h-6 w-6" />}>
+            <Skeleton className="h-[23.98px] w-[36.31px]" />
+          </Tag>
+        }
+      >
+        <Skeleton className="h-[23.98px] w-[200px]" />
```

Comment:
Question: Realised that these are some very specific values. Do we handle mobile responsiveness atm?

Reply:
thanks for the review! this will be handled as an improvement here #124